Appraisals

```diff
@@ -95,7 +95,7 @@ RAILS_VERSIONS = %w(
   4.2.5.1
   4.2.5.2
   4.2.6
-  5.0.0.beta3
+  5.0.0.1
```

Comment:
I think you should let 5.0.0 and add 4.2.7. We need to test against all Rails version to be sure it works in all cases. (Slight changes may appear between minor versions)